actions_test.go

```diff
@@ -20,6 +20,7 @@ func TestCreateWebhookAction(t *testing.T) {
 	client := BestTestClient(t, "custom_actions/create_action", testRequest)
 
 	// Act
+	// TODO: does CustomActionsWebhookActionCreateInput.LiquidTemplate work as ol.JSONString?
```

Comment:
AFAIK, the `getGraphQLType()` of `JSONString` is `JSONString` and so it wouldn't work until the API changes this to be a `JSONString`.